examples/resources/bloxone_dhcp_server/resource.tf

```diff
@@ -26,3 +26,81 @@ resource "bloxone_dhcp_server" "example_with_options" {
     }
   ]
 }
+
+data "bloxone_dns_hosts" "example_dns_host" {
+  filters = {
+    name = "DNSHost01"
+  }
+}
+
+resource "bloxone_dns_view" "example_dns_view" {
+  name = "example_dns_view"
+
+  # Other Optional fields
+  comment = "An example view"
+  tags = {
+    site = "Site A"
+  }
+
+}
+
+resource "bloxone_dns_auth_zone" "example_auth_zone" {
+  fqdn         = "domain.com."
+  primary_type = "cloud"
+
+  # Other optional fields
+  comment = "Example of an Authoritative Zone"
+  tags = {
+    site = "Site A"
+  }
+  internal_secondaries = [
+    {
+      host = data.bloxone_dns_hosts.example_dns_host.results.0.id
+    },
+  ]
+  view = bloxone_dns_view.example_dns_view.id
+}
+
+resource "bloxone_dns_auth_zone" "example_reverse_zone" {
+  comment            = "This is a comment about the resource."
+  disabled           = false
+  fqdn               = "168.192.in-addr.arpa."
+  primary_type       = "cloud"
+  gss_tsig_enabled   = false
+  initial_soa_serial = 1
+  internal_secondaries = [
+    {
+      host = data.bloxone_dns_hosts.example_dns_host.results.0.id
+    },
+  ]
+  view = bloxone_dns_view.example_dns_view.id
+}
+
+resource "bloxone_dhcp_server" "example_with_ddns" {
+  name = "example_dhcp_server_with_ddns"
+
+  #Other Optional Fields
+  comment = "dhcp server setup example for ddns"
+  tags = {
+    site = "Site A"
+  }
+
+  ddns_enabled = "true"
+  ddns_domain  = "domain.com."
+
+  # ddns_zones configuration
+  ddns_zones = [
+    {
+      gss_tsig_enabled = false
+      tsig_enabled     = false
+      tsig_key         = null
+      zone             = bloxone_dns_auth_zone.example_reverse_zone.id
+    },
+    {
+      gss_tsig_enabled = false
+      tsig_enabled     = false
+      tsig_key         = null
+      zone             = bloxone_dns_auth_zone.example_auth_zone.id
+    }
+  ]
```

Comment:
Isn't one enough?

Reply:
we thought it's good to show both primary and reverse primary. we can remove one?

Reply:
one should be enough, since there is no difference between the configs.

Reply:
done